### DISCUSSION TITLE
Change Date and Time format

### DISCUSSION THREAD

**Discussion by SodaSyrup** [valinet/ExplorerPatcher#3510] [Q&A] [answered]
Hello, maybe i didnt look good enough, but i couldnt find where i can change the Date and Time format in my system tray, is it via EP or windows settings?
Thank you for any help!

**Comment by pyrates999** [comment, marked as answer]
[that would be in the settings app.](https://www.makeuseof.com/windows-change-date-time-format/)

**Reply by SodaSyrup** [reply]
omg im blind hahaha, thank you so much!
